firmware/controllers/algo/engine_configuration.cpp

```diff
@@ -1104,11 +1047,6 @@ void setDefaultConfiguration(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 	engineConfiguration->tpsAccelLength = 12;
 	engineConfiguration->tpsAccelEnrichmentThreshold = 40; // TPS % change, per engine cycle
 
-#if EFI_PROD_CODE
```

Comment:
It should be called AFTER setDefaultBoardConfiguration(), so we move it to resetConfigurationExt().